wormchain/ibc-relayer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/golang:1.19.0@sha256:4c00329e17be6fedd8bd4412df454a205348da00f9e0e5d763380a29eb096b75
+
+#used for a readiness probe
+RUN apt-get update
+RUN apt install -y netcat
+RUN apt install -y jq
+
+WORKDIR /app
+
+RUN git clone https://github.com/nik-suri/relayer.git
```

Comment:
What's the long-term plan here?

Also, cloning a git repo isn't really deterministic.  At the very least this should be a shallow clone: `git clone --depth 1 --branch link-then-start-fix https://github.com/nik-suri/relayer.git`

Reply:
The `link-then-start` relayer command was broken and I fixed it in this PR - https://github.com/cosmos/relayer/pull/1067

The long-term plan is to use the actual relayer codebase when they make a new release.

Reply:
The point of shallow cloning is to not clone the entire history so the current clone + checkout will only work as long as your commit is the head commit of the main branch.

If we know the commit hash we can fetch a tarball at that commit and not bother with any of the git history: `curl -sSL https://github.com/cosmos/relayer/archive/c30b8d9e0ba6b45f8a576f768cad0777de545cb3.tar.gz`